README.md

```diff
@@ -32,7 +20,7 @@
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/227215951-3409f7c9-e261-465d-8666-f2dff091de17.png)

**[OPTIONAL]**

- [ ] It would be great if you make your commit messages capitalize to make your pull request professional 👍